Add `The Followpoint: Hivie, the Man of Anything and Everything` news post (#12472)
ppy/osu-wiki: news/2024/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything.md

        height: 40px;
        border-radius: 50%;
        margin-left: -50px;
    }

    .news-chat-quote__text-container {
        margin-left: 50px;
    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-lvd {
        color: #FFD1DC;
    }

    .news-chat-quote__colour-nat {
        color: #fa3703;
    }
</style>

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Hello Hivie, thank you so much for agreeing to this interview! First up, could you please give us a short introduction on who you are and what you usually do in osu! team?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>

> Hello! I'm a random person on the internet whose little corner in the world is osu!. I've been part of the [osu! team](/wiki/People/osu!_team) for the past 3 years and I've been doing a lot of things there, primarily [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leadership, osu! medals, running the Daily Challenge, and a bunch of other internal housekeeping stuff. I do also contribute to other areas like [Project Loved](/wiki/Community/Project_Loved), [Tournament Committee](/wiki/People/Tournament_Committee), [MBC](/wiki/Contests/Monthly_Beatmapping_Contest), [osu! wiki](/wiki/osu!_wiki) maintenance, and so on.

Reviewer: either way works

What do you like about developing them the most?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>

osu! medal hunters are insane schizos (in a good way), and trying to match that energy when brainstorming medals is always fun.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Speaking of development, do you have any ETA on when medals will display which map you received them on?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>

> Honestly I'm not sure, It might go against the spirit of secrecy with Hush-Hush medals, so it will need further considerations by the osu! team.

Author: ```suggestion
Honestly I'm not sure, It might go against the spirit of secrecy with Hush-Hush medals, so it will need further consideration by the osu! team.
```

---
layout: post

> title: "The Followpoint: Hivie, the Man of Anything and Everything"

Reviewer 2: I kinda agree it reads wrong.. "Ever heard of the person who does literally anything you can think of?"

hivie does a great job around the place, but i wouldn't go as far as saying "literally anything and everything", especially when only four roles are touched on.

i'm not trying to downplay their role, but it may read a bit odd to others who are involved in a similar number of community ventures.

> title: "The Followpoint: Hivie, the Man of Anything and Everything"

Author: Tbh yea, I admit I went clickbait mode here, so if you have any good ideas - I'll gladly change the title. But like I said, the idea is to tell people Hivie does a lot, so I will not agree to something like `osu!taiko Paragon`

> title: "The Followpoint: Hivie, the Man of Anything and Everything"

Author: @Walavouchey @peppy What do you think of "Hivie, the Powerhouse Behind the Scenes"?

> title: "The Followpoint: Hivie, the Man of Anything and Everything"

Reviewer 2: Dunno. @Ephemeralis might be able to give a suiting one here since he previously had kinda the same role.

Welcome or welcome back to The Followpoint, a series where we interview various players and community figures to get to know them better! (Note: Please pretend this news post released on November 10) Today is a special day for us, it's our first anniversary! And we just had to chose a special interviewee for today, both for us and osu! community in general, [Hivie](https://osu.ppy.sh/users/14102976)!

Hivie is an [osu!team](/wiki/People/osu!_team) member responsible for community affairs, [NAT (Nomination Assessment Team)](/wiki/People/Nomination_Assessment_Team) leader, [Project Loved](/wiki/Community/Project_Loved) manager, [medals](/wiki/Medals) developer and works on even more things than I listed here, so you likely heard his name at least once, no matter the aspect of osu! you are interested in! Funny enough, [our first interview](https://osu.ppy.sh/home/news/2023-11-10-the-followpoint-ephemeral-the-man-behind-the-medals) was with [Ephemeral](https://osu.ppy.sh/users/102335), who also was a community maintainer and medals developer. That, and the fact Hivie often helps us with polishing our interviews makes him a perfect interviewee for today! 

> This interview will looks slightly different from the ones you got used to, as we'd like to make it in the spirit of our first one. That means it will be less flow-based, but the questions will still be split into categories.  

Reviewer 3: ```suggestion
This interview will look slightly different from the ones you got used to, as we'd like to make it in the spirit of our first one. That means it will be less flow-based, but the questions will still be split into categories.  
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>

the osu! community has been, and still is, a once in a lifetime experience for me, and I've grown attached to all the workflows and people I've met along the way. The reason I'm still here, and will be for the foreseeable future, is because it's a place that I can call *home* in some form, despite the ups and downs.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

What would you call the greatest gift that osu! has given you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>

I want to name 2 things:

> 1. People skills: I've been a pretty timid person for a good chunk for my life, and osu! has helped me develop my social skills in a way I didn't expect, both through working together on projects, and through managing and leading teams.

Reviewer 3: ```suggestion
1. Personal development: I've been a pretty timid person for a good chunk for my life, and osu! has helped me develop my social skills in a way I didn't expect, both through working together on projects, and through managing and leading teams.
```
"people skills" looks odd to me :thinking:

Author: Would prefer to keep it as is since it's a real term https://en.wikipedia.org/wiki/People_skills

- If mapping or modding isn't your jam, consider looking into community projects, such as [tournaments](https://osu.ppy.sh/community/forums/55), or join the [osu! Discord server](https://discord.com/invite/ppy) to keep up with the latest official and community projects!
- If you're more of a technical person, hone your skills via the following:
  - Be a part of the game's [client](https://github.com/ppy/osu) or [website](https://github.com/ppy/osu-web) development.
  - Help out with [maintaining the osu! wiki](/wiki/osu!_wiki/Contribution_guide).
  - Head to the `#osu-news` channel in the osu! Discord server and take a jab at news coverage by writing articles on new ideas, or contributing to existing news projects.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

> Some players believe there is a lack of communication from osu!'s side in different situations at times, which makes them trust osu! team less, what do you think about that and is the team planning to or already doing something to fix that?

Reviewer 2: Let's just drop this question and answer from the interview to keep things simple.

> Some players believe there is a lack of communication from osu!'s side in different situations at times, which makes them trust osu! team less, what do you think about that and is the team planning to or already doing something to fix that?

Reviewer 2: As for "ghosting", I'd love to follow up and provide you with a resolution. 95% of the time I look into these cases there's very good reason for it. You can send me an email at pe@ppy.sh if you'd like.

Hivie is an [osu! team](/wiki/People/osu!_team) member, but is also a [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leader, [Project Loved](/wiki/Community/Project_Loved) manager, [medal](/wiki/Medals) coordinator, among so many other things. If you've been active in mapping or development discussion over the past years, you've surely seen him around! Funnily enough, [our first interview](https://osu.ppy.sh/home/news/2023-11-10-the-followpoint-ephemeral-the-man-behind-the-medals) was with [Ephemeral](https://osu.ppy.sh/users/102335), who also was a community maintainer and medal developer. That, and the fact that Hivie often helps us with polishing our interviews, makes interviewing him that much more fitting for today!

This interview will look slightly different from the ones you got used to, as we'd like to make it in the spirit of our first one. That means it will be less flow-based, but the questions will still be split into categories.  

> I'd also like to say something at the end of this interview, but first things first, let's see what Hivie has to say! :3

Reviewer 2: Could probably cut this line, or at least remove the smiley. Adding the outro is fine but I don't think you need to mention it here.

Reviewer: the line itself makes sense to keep since it eases the reader into the outro's more personal tone shift, so i've just removed the smiley

        margin-left: 50px;
    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-lvd {
        color: #FFD1DC;
    }

    .news-chat-quote__colour-nat {
        color: #fa3703;
    }
    
    .news-chat-quote__text-container ol,
    .news-chat-quote__text-container ul {
        margin-bottom: 25px;    
    }
</style>

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Hello Hivie, thank you so much for agreeing to this interview! First up, could you please give us a short introduction on who you are and what you usually do in osu! team?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>

> Hello! I'm a random person on the internet whose little corner in the world is osu!. I've been part of the [osu! team](/wiki/People/osu!_team) for the past 3 years and I've been doing a lot of things there, primarily [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leadership, osu! medals, running the Daily Challenge, and a bunch of other internal housekeeping stuff. I do also contribute to other areas like [Project Loved](/wiki/Community/Project_Loved), [Tournament Committee](/wiki/People/Tournament_Committee), [MBC](/wiki/Contests/Monthly_Beatmapping_Contest), [osu! wiki](/wiki/osu!_wiki) maintenance, and so on.

Reviewer 4: ```suggestion
Hello! I'm a random person on the internet whose little corner in the world is osu!. I've been part of the [osu! team](/wiki/People/osu!_team) for the past 3 years and I've been doing a lot of things there, primarily [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leadership, osu! medals, running the [Daily Challenge](/wiki/Gameplay/Daily_challenge), and a bunch of other internal housekeeping stuff. I do also contribute to other areas like [Project Loved](/wiki/Community/Project_Loved), [Tournament Committee](/wiki/People/Tournament_Committee), [MBC](/wiki/Contests/Monthly_Beatmapping_Contest), [osu! wiki](/wiki/osu!_wiki) maintenance, and so on.
```
